SPECS/packer/packer.spec

```diff
@@ -1,7 +1,10 @@
+%global debug_package %{nil}
+%define our_gopath %{_topdir}/.gopath
+
 Summary:        Tool for creating identical machine images for multiple platforms from a single source configuration.
 Name:           packer
-Version:        1.10.1
-Release:        2%{?dist}
+Version:        1.9.5
```

Comment:
Won't this need an epoch bump to force using this older version? Otherwise `dnf`/`tdnf` will prefer `packer-1.10.1` when querying the package repo

Reply:
That's a good question. We need to revert the newer version anyway, so the epoch shouldn't be needed. But it won't hurt, since we're it's 2.0-only and it'll shield us from any "transition period" errors people may hit, if we tried to sync removal and publishing without the epoch bump. I'll bump in a minute.